crates/control/src/rule_obstacle_composer.rs

```diff
@@ -23,8 +23,7 @@ pub struct CycleContext {
         RequiredInput<Option<FilteredGameControllerState>, "filtered_game_controller_state?">,
     ball_state: Input<Option<BallState>, "ball_state?">,
 
-    center_circle_obstacle_increase:
-        Parameter<f32, "rule_obstacles.center_circle_obstacle_increase">,
+    center_circle_obstacle_offset: Parameter<f32, "rule_obstacles.center_circle_obstacle_offset">,
```

Comment:
`offset` sounds like moving the obstacle. You want to add some additional delta to the radius. The `..._increase` was kinda fine to me. Maybe keep it? Or find an even better name

Reply:
Renamed